### DISCUSSION TITLE
Delay between commit and release

### DISCUSSION THREAD

**Discussion by HazMatt69** [jlesage/docker-handbrake#353] [Q&A] [answered]
Hi,

What is the reason for the delay (from 2 to 5 days) between commit and release? Is that for final testing?
Thanks!

**Comment by vincejv** [comment]
@HazMatt69 are you able to pull Handbrake 1.8.0 from docker hub?

**Reply by HazMatt69** [reply]
Nope!

**Reply by vincejv** [reply]
Tried building it from the git repo, it's unstable, video previews don't work, presets are buggy, it's not saved, old ones are not read, there's a GTK warning spamming the Activity logs, it's unstable at the moment and I guess the baseimage https://github.com/jlesage/docker-baseimage needs to be updated for GTK 4 and Handbrake 1.8 compatibility.

**Reply by HazMatt69** [reply]
Yeah, I'm pretty sure it's still in testing...

**Reply by jlesage** [reply, marked as answer]
Yes, some time is needed to fix any compilation issues and any problems encountered with the new release.

**Comment by HazMatt69** [comment]
@jlesage Are you waiting for 1.8.1 to release a new version?

**Reply by vincejv** [reply]
i think the issue is there are bugs and glitches running GTK4 apps/or handbrake 1.8 on docker and she needs time to figure things out, PRs should be welcome.

**Reply by jlesage** [reply]
No, not waiting for 1.8.1 :)
Currently, the only issue that seems to remain is the live preview that hangs.
